toolkit/data-sources/db-sync/src/bridge/mod.rs

```diff
@@ -10,6 +10,36 @@
 //! at the illiquid supply address can only spend UTXOs from outside of it. This guarantees
 //! that the observability layer can always correctly identify the number of tokens transfered
 //! by calculating the delta of `tokens in the new UTXO` - `tokens in the old ICS UTXOs`.
+//!
+//! # Usage
+//!
+//! ```rust
+//! use partner_chains_db_sync_data_sources::*;
+//! use sqlx::PgPool;
+//! use std::{ error::Error, sync::Arc };
+//!
+//! // number of block ahead the bridge data source should cache
```

Comment:
Could you please add here that it is used for syncing and we data source never caches non-final blocks data?